erpnext/stock/report/stock_balance/stock_balance.js

```diff
@@ -53,13 +53,14 @@ frappe.query_reports["Stock Balance"] = {
 			"width": "80",
 			"options": "Warehouse",
 			get_query: () => {
-				var warehouse_type = frappe.query_report.get_filter_value('warehouse_type');
-				if(warehouse_type){
-					return {
-						filters: {
-							'warehouse_type': warehouse_type
-						}
-					};
+				let warehouse_type = frappe.query_report.get_filter_value("warehouse_type");
+				let company = frappe.query_report.get_filter_value("company");
+
+				return {
+					filters: {
+						...(warehouse_type) && {warehouse_type},
+						...(company) && {company}
```

Comment:
```suggestion
						...warehouse_type && {warehouse_type},
						...company && {company}
```
here too parens seem unnecessary; operator precedence takes care if it ig